app/lib/dttp/mappable.rb

```diff
@@ -30,6 +30,10 @@ def degree_type_id(degree_type)
       Dttp::CodeSets::DegreeTypes::MAPPING.dig(degree_type, :entity_id)
```

Comment:
Can we remove the reference `Dttp::` in other places since it's not needed?

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/564/commits/00eba96ddd2d717211ed74abda9f91646283183d